build.sbt

```diff
@@ -242,6 +243,14 @@ lazy val cliTest = project
     testkit
   )
 
+lazy val gui = project
+  .in(file("app/gui"))
+  .settings(CommonSettings.prodSettings: _*)
+  .dependsOn(
+    cli,
```

Comment:
So why do we have both the `cli` and the `wallet` as a dep? Shouldn't we be using the wallet functionality via the `cli`? Is this because limitations to the current `WalletRoutes` where things we need are not exposed? If so please file an issue